HW_1/task_6.py

```diff
@@ -0,0 +1,23 @@
+# Задача 2 домашнего задания Семинара1
+# Программа загадывает число от 0 до 1000.
+# Необходимо угадать число за 10 попыток.
+# Программа должна подсказывать «больше» или «меньше» после каждой попытки
+import random
+
+number = random.randint(0, 1000)
+tries = 10
+
+print("Угадайте число от 0 до 1000. У вас есть 10 попыток.")
+
+for i in range(tries):
+    guess = int(input("Попытка №{}: ".format(i + 1)))
+    if guess == number:
+        print("Вы угадали число!")
+        break
+    elif guess < number:
+        print("Загаданное число больше")
+    else:
+        print("Загаданное число меньше")
+
+if guess != number:
+    print("Вы не угадали число. Было загадано", number)
```

Comment:
Если попытки закончили, то очевидно, то число не угадано.
```suggestion
else:
    print("Вы не угадали число. Было загадано", number)
```